client/js/options.js

```diff
@@ -44,7 +44,7 @@ userOptions = null;
 module.exports = options;
 
 module.exports.shouldOpenMessagePreview = function(type) {
-	return (options.links && type === "link") || (options.thumbnails && type === "image");
+	return (options.links && type === "link") || (options.media && (type === "image" || type === "audio"));
```

Comment:
Off-topic/Not for this PR: @xPaw, @MaxLeiter, what about merging both under `options.preview` or something?

Reply:
I think there's a slight difference, in that links will be low-bandwidth, but media could vary. At least, I think that's how @xPaw and I see it at the moment.

Reply:
That's fair, thanks!